msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -81,294 +45,130 @@ Following is a preview local image:
 
 <b><summary>Preview manifest file in remote environment</b></summary>
 
-To preview manifest file using Visual Studio Code:
-
-* Select **Provision in the cloud** under **DEPLOYMENT** in Teams Toolkit.
-  
-   :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/provision.png" alt-text="Screenshot is an example of showing the selection of provision in the cloud resource.":::
-
-* To preview the manifest file using command palette. You can trigger **Teams: Provision in the cloud** from command palette.
-
-   :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/command palatte.png" alt-text="Screenshot is an example of showing provision cloud resource using command palette.":::
+To preview manifest file in remote environment, you can trigger `Provision` from tree view or `Teams: Provision in the cloud` from command palette. It generates environment variables for remote Teams app, build app package and preview manifest under `appPackage/build` folder.
 
-It generates configuration for remote Teams app, builds package, and preview manifest under `build/appPackage` folder.
+You can also trigger Zip Teams App Package from tree view or `Teams: Zip Teams app Package` from command palette to generate preview manifest and app package.
 
-You can preview manifest file in remote environment by two methods:
+:::image type="content" source="../assets/images/teams-toolkit-v2/customize app manifest/zip-app-package (1).png" alt-text="Screenshot showing the selection of zip Teams app package in preview manifest file.":::
 
-* Use preview option in CodeLens.
-* Use **Zip Teams metadata package** option.
-
-To preview manifest file by using preview option in CodeLens:
-
-1. Select **Preview** in the CodeLens of `manifest.template.json` file.
-
-   :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/preview-23.png" alt-text="Screenshot is an example of showing preview in the CodeLens of manifest file." lightbox="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/preview-23.png":::
-
-1. Select your environment.
+## Customize Teams app manifest for Visual Studio Code
```

Comment:
This section and below should not be collapsed by "Preview manifest file in remote environment"

Reply:
Updated.